amplify.yml

```diff
@@ -7,6 +7,7 @@ frontend:
     build:
       commands:
         - rm -rf .next/
+        - cp .env.$USER_BRANCH .env
```

Comment:
this. Without this prod would not work because the `.env` file contains development/staging variables